app/src/main/java/com/kaamchor/android/fameofshame/CountDownActivity.java

```diff
@@ -0,0 +1,40 @@
+package com.kaamchor.android.fameofshame;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.view.Window;
+import android.widget.NumberPicker;
+
+public class CountDownActivity extends AppCompatActivity {
+
+    int hr = 0;
+    int min = 0;
```

Comment:
private static final int for constants.
private as we should always keep visibility to the lowest possible as general practice.